tests/value/__init__.py

```diff
@@ -59,11 +63,19 @@ def check_values(
     :param atol: absolute tolerance of elements in `values` with respect to
         elements in `exact_values`. E.g. if atol = 0.1, and rtol = 0 we must
         have |value - exact_value| < 0.1 for every value.
+    :param extra_values_names: Sequence of names of extra values that should
+        also be compared.
     """
     values.sort()
     exact_values.sort()
 
     assert np.allclose(values.values, exact_values.values, rtol=rtol, atol=atol)
+    if extra_values_names is None:
+        return
+    for name in extra_values_names:
```

Comment:
Maybe default the attribute to an empty tuple in order to avoid the None check

Reply:
Done.